material-design-catalog.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Material Design Catalog
+extra_css:
+  - /css/catalog.css
+sidebar: home_sidebar
+permalink: /material-design-catalog/
+---
+
+This page catalogs Flutter’s reference implementation of UI components from Google’s [Material Design Specification](https://material.google.com/). You can see many of the material design widgets in action in our [Flutter Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery) demo app. For an overview of different kinds of widgets in Flutter, please take a look at the [Basic Widgets Overview](/basic-widgets/).
```

Comment:
Should we say something about how complete this list is?

For example, are we claiming "these are all the widgets" ? Or, something like, "for the complete list, check out __________"

I like the later because we might add a widget and forget to update this page.